resources/content/config-content/dns/content/etc/dns/answers.json.ftl

```diff
@@ -0,0 +1,19 @@
+{
+    "default": {
+        "recurse": ["8.8.8.8"],
+        "rancher.com.": ["1.2.3.4"]
```

Comment:
`rancher.com.` was just an example & should be removed.  The recurse addresses should probably be pulled from /etc/resolv.conf or something instead of being hardcoded to Google.